src/client/pages/NewAccountReview.tsx

```diff
@@ -0,0 +1,174 @@
+import { Consent } from '@/shared/model/Consent';
+import { ToggleSwitchInput } from '../components/ToggleSwitchInput';
+import {
+	InformationBox,
+	InformationBoxText,
+} from '../components/InformationBox';
+import { ExternalLink } from '../components/ExternalLink';
+import locations from '@/shared/lib/locations';
+import { palette, space, textSans } from '@guardian/source-foundations';
+import { css } from '@emotion/react';
+import { Button } from '@guardian/source-react-components';
+import { buildUrlWithQueryParams } from '@/shared/lib/routeUtils';
+import { CmpConsentedStateHiddenInput } from '../components/CmpConsentStateHiddenInput';
+import { CsrfFormField } from '../components/CsrfFormField';
+import { QueryParams } from '@/shared/model/QueryParams';
+
+const consentToggleCss = css`
+	display: flex;
+	margin-top: ${space[6]}px;
+	margin-bottom: ${space[4]}px;
+	flex-direction: column;
+	gap: ${space[3]}px;
+`;
+
+const switchRow = css`
+	border: 0;
+	padding: 0;
+	margin: 0;
+	${textSans.medium()}
+	border-radius: 4px;
+	border: 1px solid ${palette.neutral[38]};
+	padding: ${space[2]}px;
+`;
+
+const labelStyles = css`
+	justify-content: space-between;
+	& > span:first-of-type {
+		color: ${palette.neutral[20]};
+		${textSans.xsmall({ fontWeight: 'bold' })}
+	}
+	& > span:last-of-type {
+		align-self: flex-start;
+		color: ${palette.neutral[46]};
+		${textSans.xsmall()}
+	}
+`;
+
+const subheadingStyles = css`
+	margin-top: ${space[6]}px;
+	margin-bottom: ${space[1]}px;
+	${textSans.medium({ fontWeight: 'bold' })};
+`;
+
+const listStyles = css`
+	margin-top: 0;
+	padding-left: 1rem;
+`;
+
+export interface NewAccountReviewProps {
+	profiling?: Consent;
+	advertising?: Consent;
+	queryParams: QueryParams;
+	hasCmpConsent: boolean;
+}
+
+export const NewAccountReview = ({
+	profiling,
+	advertising,
+	queryParams,
+	hasCmpConsent,
+}: NewAccountReviewProps) => {
+	if (!profiling && !advertising) {
+		return (
+			<MainLayout pageHeader="You're signed in! Welcome to the Guardian.">
+				<form
+					action={buildUrlWithQueryParams('/welcome/review', {}, queryParams)}
+					method="post"
+				>
+					<CmpConsentedStateHiddenInput cmpConsentedState={hasCmpConsent} />
+					<CsrfFormField />
+					<Button css={buttonStyles({})} type="submit" priority="primary">
+						Continue to the Guardian
+					</Button>
+				</form>
+			</MainLayout>
+		);
+	}
+	return (
+		<MainLayout pageHeader="You're signed in! Welcome to the Guardian.">
+			<MainBodyText>
+				Before you start, confirm how you’d like the Guardian to use your
+				signed-in data.
+			</MainBodyText>
+			<form
+				action={buildUrlWithQueryParams('/welcome/review', {}, queryParams)}
+				method="post"
+				// onSubmit={({ target: form }) => {
+				// 	onboardingFormSubmitOphanTracking(
+				// 		page,
+				// 		pageData,
+				// 		// have to explicitly type as HTMLFormElement as typescript can't infer type of the event.target
+				// 		form as HTMLFormElement,
+				// 	);
+				// }}
```

Comment:
whats this commend about?

do we need to enable this for tracking or remove it?